src/Uno.UI.Tests/App/Xaml/Test_MarkupExtension.xaml

```diff
@@ -47,6 +48,16 @@
 
 		<TextBlock x:Name="Text10"
 				   DataContext="{ex:NoReturnTypeMarkupExt UseValue=UseValue1, Value1={StaticResource TestEntity}}" />
+
+		<TextBlock x:Name="Text11"
+				   Text="{ex:SimpleMarkupExtExtesion TextValue='Just a simple ... fullname'}" />
+
+		<TextBlock x:Name="Text12"
+				   Text="{ex:SimpleMarkupExtExtesion TextValue={StaticResource ResourceStringFull}}" />
+
+		<TextBlock x:Name="Text13"
+				   beh:AttachedPropertiesBehavior.CustomText="{ex:SimpleMarkupExt TextValue='String from attached property Fullname'}" />
```

Comment:
Is this a typo or did you want to use `SimpleMarkup` instead ?

Reply:
is typo